anonymizer/base.py

```diff
@@ -172,6 +178,24 @@ def source():
             source = self.faker.lorem
         return self.get_allowed_value(source, field)
 
+
+    def unique_lorem(self, field=None, val=None):
+        """
+        Returns lorem ipsum text guaranteed to be unique. First uses lorem function
+        then adds a unique integer suffix.
+        """
+        lorem_text = self.lorem(field, val)
+        max_length = getattr(field, 'max_length', None)
```

Comment:
this max_length may be `None`, and then later u seem to use it as part of `lorem_text + suffix_str)[-max_length:]`
is that really gonna work out?